sos/cleaner/__init__.py

```diff
@@ -685,6 +686,11 @@ def obfuscate_report(self, archive):  # pylint: disable=too-many-branches
                         archive.should_remove_file(short_name)):
                     archive.remove_file(short_name)
                     continue
+                if (self.opts.keep_binary_files and
+                        (file_is_binary(fname) or
+                         archive.should_remove_file(short_name))):
+                    archive.should_skip_file(short_name)
```

Comment:
This line is redundant; `should_skip_file` is just a no-op test, no reason to call it as a command.

Reply:
Ack, I removed it in the next version